### DISCUSSION TITLE
AppThemeBinding

### DISCUSSION THREAD

**Discussion by jfraschilla** [dotnet/maui#7571] [Q&A]
I get a runtime exception using AppThemeBinding on a ImageButton Source property.  Is the following valid:

`<ImageButton
    Grid.Row="1"
    Margin="10"
    Command="{Binding CreateCommand}"
    Source="{AppThemeBinding Light=add_box_dark_48dp.svg, Dark=add_box_light_48dp.svg}"
    HorizontalOptions="End"
    VerticalOptions="End"/>
`

**Comment by J-Swift** [comment]
If you are using Resizetizer (i.e. the thing that handles `MauiImage` tags in your `csproj`) then you reference the images as a `png`, not `svg`.

https://docs.microsoft.com/en-us/dotnet/maui/user-interface/images/images
